glean-core/android/src/test/java/mozilla/telemetry/glean/net/BaseUploaderTest.kt

```diff
@@ -18,9 +18,7 @@ class BaseUploaderTest {
     private val testPath: String = "/some/random/path/not/important"
     private val testPing: String = "{ 'ping': 'test' }"
     private val testHeaders: HeadersList = mutableListOf(Pair("X-Test-Glean", "nothing-to-see-here"))
-    private val testDefaultConfig = Configuration().copy(
-        userAgent = "Glean/Test 25.0.2"
-    )
+    private val testDefaultConfig = Configuration().copy()
```

Comment:
```suggestion
    private val testDefaultConfig = Configuration()
```

No need to copy: copy is useful when overriding specific fields :)